submissions/adeyemiJuwon/HTMLforms/signup.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Account create</title>
+</head>
+<body>
+  <header>
+    <div>
+        <input type="search" name="search" value="" aria-label="Search" placeholder="Search">
+            <ul>
+                <li>
+                    <a href="login.html">Log in</a>
```

Comment:
You don't have a login page. You need to update this. @Samju200

Reply:
You told me to remove the login page just now